slsSupportLib/include/ServerInterface2.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "DataSocket.h"
+namespace sls {
+class ServerInterface2;
```

Comment:
Will you be removing ServerInterface when this is merged?

Reply:
Yes, the implementation of ServerInterface will move to the new class.